build/upgrade_highlightjs.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Run with specific branch/tag (e.g. ./upgrade_highlightjs.sh 10.6.0) or with no arguments to use master
+
+# TODO: This list of languages should be made more dynamic based on the file extensions Preview supports, see src/lib/extensions.js
```

Comment:
Should we include a comment in `src/lib/extensions.js` that if the file types change we need to regenerate the highlightjs bundle?

Reply:
Good idea, I'll add a comment there